lib/tests/dataLayer.test.js

```diff
@@ -136,14 +136,14 @@ describe("Submit Form data", () => {
       .toBeCalledTimes(1)
       .toBeCalledWith(
         expect.objectContaining({
-          pathname: "/en/confirmation",
+          pathname: "/id/1/confirmation",
           query: {
             htmlEmail: undefined,
             referrerUrl: expect.any(String),
             pageText: expect.any(String),
           },
         }),
-        expect.objectContaining({ pathname: "/en/confirmation" })
+        expect.objectContaining({ pathname: "/id/1/confirmation" })
```

Comment:
In order to support the language rendering this would need to check for `"/en/id/1/confirmation"`.